### PR TITLE
fix: do not try to filter svg images

### LINKED ISSUE #695: product-latest Unable to open image svg
If I request the endpoint `http://localhost/shop-api/product-latest/` I got the flowing error
```
{
    "code": 500,
    "message": "Unable to open image /srv/sylius/public/media/image/43/97/e0fe3450b896642e5efb574872a3.svg"
}
```
Steps to reproduce:
1. Clone https://github.com/Sylius/Sylius-Standard
2. docker-compose up
3. install sylius with sample data
4. install shopapiplugin
5. request the endpoint `http://localhost/shop-api/product-latest/`

As far as I understand svg is with embeded png
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   version="1.1"
   id="svg10"
   width="1200"
   height="1200"
   viewBox="0 0 1200 1200"
   sodipodi:docname="jeans.svg"
   inkscape:version="1.0.1 (3bc2e813f5, 2020-09-07)">
  <metadata
     id="metadata16">
    <rdf:RDF>
      <cc:Work
         rdf:about="">
        <dc:format>image/svg+xml</dc:format>
        <dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
        <dc:title></dc:title>
      </cc:Work>
    </rdf:RDF>
  </metadata>
  <defs
     id="defs14" />
  <sodipodi:namedview
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1"
     objecttolerance="10"
     gridtolerance="10"
     guidetolerance="10"
     inkscape:pageopacity="0"
     inkscape:pageshadow="2"
     inkscape:window-width="743"
     inkscape:window-height="480"
     id="namedview12"
     showgrid="false"
     inkscape:zoom="0.17280891"
     inkscape:cx="600"
     inkscape:cy="800"
     inkscape:window-x="2446"
     inkscape:window-y="209"
     inkscape:window-maximized="0"
     inkscape:current-layer="g18" />
  <g
     inkscape:groupmode="layer"
     inkscape:label="Image"
     id="g18">
    <image
       width="1200"
       height="1600"
       preserveAspectRatio="none"
       xlink:href="data:imag
...
```
Sample image

[e0fe3450b896642e5efb574872a3.zip](https://github.com/Sylius/ShopApiPlugin/files/5839254/e0fe3450b896642e5efb574872a3.zip)

### PR DESCRIPTION
**Pull request by EmilMassey**
Fixes #695 
Liip Imagine cannot handle svg files so we keep original svg path

### CODE REVIEW

**Comment by lchrusciel** [comment]
Thanks, Emil! :tada:

File: src/Factory/ImageViewFactory.php [hunk 1]
```diff
@@ -37,8 +37,17 @@ public function create(ImageInterface $image): ImageView
...
+        } else {
+            $imageView->cachedPath = $image->getPath();
+        }
 
         return $imageView;
     }
+    
+    private function canImageBeFiltered(string $path): bool
+    {
+        return substr($path, -4) !== '.svg';
```
Comment (FDiskas) [inline src/Factory/ImageViewFactory.php:51]: Why not?
```php
pathinfo($file, PATHINFO_EXTENSION);
```

Comment (EmilMassey) [reply src/Factory/ImageViewFactory.php:51]: I decided to use exactly the same logic as in Core Sylius: https://github.com/Sylius/Sylius/blob/fa4c1d30dd9f94f4c8bb4e80b8fb3208fdf38c62/src/Sylius/Bundle/CoreBundle/Twig/FilterExtension.php#L46

Comment (FDiskas) [reply src/Factory/ImageViewFactory.php:51]: Thanks for the fix

**Pull request merged**